docs/src/VariableList.md

```diff
@@ -0,0 +1,84 @@
+# CliMA Variable List: 
+
+This document is currently for collaborators within the project with access to the Overleaf CliMA-Atmos docs. The purpose of this page is to unify the naming conventions used in the Overleaf document in a manner useful for coding. This document suggests 'reserved' variable names in <property>_<species> format with the default working fluid (no-subscript) being moist air. Contributors to the CliMA repository are welcome to suggest changes when necessary.
+
+### Names reserved for 'use and throw' / debug variables
+```
+dummy
+scratch
+```
+
+### 2.1  Working Fluid and Equation of State
+```
+q_dry = dry air mass fraction
+q_vap = specific humidity, vapour
+q_liq = specific humidity, liquid
+q_ice = specific humidity, ice
+q_con = specific humidity, condensate
+q_tot = specific humidity, total
```

Comment:
We have `q_i`, `q_l` etc. in the thermodynamics code. But I like the clarity of the conventions you suggest. Please feel free to change globally.